cwms-aaa-client/src/main/java/mil/army/usace/hec/cwms/aaa/client/CwmsLoginController.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2022
+ * United States Army Corps of Engineers - Hydrologic Engineering Center (USACE/HEC)
+ * All Rights Reserved.  USACE PROPRIETARY/CONFIDENTIAL.
+ * Source may not be released without written approval from HEC
+ */
+
+package mil.army.usace.hec.cwms.aaa.client;
+
+import com.fasterxml.jackson.core.json.JsonReadFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import java.io.IOException;
+import mil.army.usace.hec.cwms.http.client.ApiConnectionInfo;
+import mil.army.usace.hec.cwms.http.client.EndpointInput;
+import mil.army.usace.hec.cwms.http.client.HttpRequestBuilder;
+import mil.army.usace.hec.cwms.http.client.HttpRequestBuilderImpl;
+import mil.army.usace.hec.cwms.http.client.HttpRequestResponse;
+import mil.army.usace.hec.cwms.http.client.SslCanceledException;
+
+public final class CwmsLoginController {
+
+    private static final String LOGIN_ENDPOINT = "login";
+    private static final String LOGOUT_ENDPOINT = "logout";
+    private static final String DOD_AGREE_PARAMETER = "dod_agree";
+    private static final String JSESSIONID = "JSESSIONID";
```

Comment:
JESSIONIDSSO. Otherwise the session specific to CWMSLogin and not the instance of tomcat.

... actually probably should store and transmit both.

Reply:
added separate jsessionidSso field